### ISSUE TITLE
emonCMS version

### ISSUE THREAD

**Issue by stuartpittaway** [frnandu/diypowerflow#1]
This is really nicely done - have you tried to convert this into an emonCMS app ?

**Comment by frnandu** [comment]
Yes, actually first I did a emoncms dashboard : https://emoncms.org/dashboard/view&id=39520
But then I wanted some animation so ended up doing JS +SVG to anchieve this.
I'm not sure how to make emoncms apps (not dashboard), and if it is possible to make such animation there. Do you have any experience with them?

**Comment by stuartpittaway** [comment]
The apps are just javascript - I wrote the CostComparison one and I did start doing something similar to the Tesla one - link below.

https://community.openenergymonitor.org/t/new-energy-flow-app-in-development/5224

However I lost the source code after a disk crash!

**Comment by frnandu** [comment]
If you want to try to adapt my code to an emoncms app, go for it.
If not I will try to do it, when I have a bit of spare time.
